.github/actions/build-platform-docker/action.yml

```diff
@@ -71,10 +72,12 @@ runs:
       id: get-date
       shell: bash
       run: echo "result=$(date --iso-8601=seconds)" >> $GITHUB_OUTPUT
-    - name: Set image name from params and/or project version
+    - name: Build image names from params and/or project version
       id: get-image
-      shell: bash
-      run: echo "result=${{ inputs.repository }}:${{ inputs.version || steps.get-version.outputs.result }}" >> $GITHUB_OUTPUT
+      uses: docker/metadata-action@v5
+      with:
+        images: ${{ inputs.repository }}
+        tags: ${{ inputs.version || steps.get-version.outputs.result }}
```

Comment:
@houssain-barouni I think this change broke zeebe chaos and e2e tests on main. We are setting the env `IMAGE` in the workflow `zeebe-testbench.yaml` as follows
```
IMAGE: ${{ steps.build-zeebe-docker.outputs.image }}
```
But it's value is empty. Could you please advice how to fix this?

Reply:
The problem is tracked in https://github.com/camunda/camunda/issues/20001 and I am working on a fix in https://github.com/camunda/camunda/pull/20087

Reply:
https://github.com/camunda/camunda/pull/20087 got merged so hopefully Zeebe chaos and E2E tests on `main` work again